src/app/(post-login)/submission/program/[shortName]/clinical-data/ClinicalEntityDataTable.tsx

```diff
@@ -636,71 +638,159 @@ const ClinicalEntityDataTable = ({
+			background-color: ${theme.colors.grey_4};
+			border-bottom: 1px solid ${theme.colors.grey_2};
+			border-right: 1px solid ${theme.colors.grey_2};
+			padding: 5px;
+			font-size: 13px;
+			text-align: left;
+		`;
+
+		const stickyCSS = css`
+			position: sticky !important;
+			left: 0;
+			top: 0;
+			z-index: 1;
+			background-color: white;
+		`;
+
 		columns = [
 			{
 				id: 'clinical_core_completion_header',
-				header: (
-					<div
-						css={css`
-							display: flex;
-							align-items: center;
-							justify-content: center;
-							position: relative;
-						`}
-					>
-						CLINICAL CORE COMPLETION
-						<Tooltip
-							style={{ position: 'absolute', left: 'calc(100% - 20px)', top: '-2px' }}
-							html={
-								<p
-									css={css`
-										margin: 0px;
-										margin-right: 6px;
-									`}
-								>
-									For clinical completeness, each donor requires: <br />
-									DO: at least one Donor record <br />
-									PD: at least one Primary Diagnosis record <br />
-									NS: all the registered Normal DNA Specimen record <br />
-									TS: all the registered Tumour DNA Specimen record <br />
-									TR: at least one Treatment record <br />
-									FO: at least one Follow Up record <br />
-								</p>
-							}
+				meta: { customHeader: true },
+				sortingFn: sortEntityData,
+				header: (props) => {
+					return (
+						<th
+							colSpan={props.colSpan}
+							css={css`
+								${headerStyle};
+								border-right: 3px solid ${theme.colors.grey};
+							`}
 						>
-							<Icon name="question_circle" fill="primary_2" width="18px" height="18px" />
-						</Tooltip>
-					</div>
-				),
+							<div
+								css={css`
+									display: flex;
+									align-items: center;
+									justify-content: center;
+									position: relative;
+								`}
+							>
+								CLINICAL CORE COMPLETION
+								<Tooltip
+									style={{ position: 'absolute', left: 'calc(100% - 20px)', top: '-2px' }}
+									html={
+										<p
+											css={css`
+												margin: 0px;
+												margin-right: 6px;
+											`}
+										>
+											For clinical completeness, each donor requires: <br />
+											DO: at least one Donor record <br />
+											PD: at least one Primary Diagnosis record <br />
+											NS: all the registered Normal DNA Specimen record <br />
+											TS: all the registered Tumour DNA Specimen record <br />
+											TR: at least one Treatment record <br />
+											FO: at least one Follow Up record <br />
+										</p>
+									}
+								>
+									<Icon name="question_circle" fill="primary_2" width="18px" height="18px" />
+								</Tooltip>
+							</div>
+						</th>
+					);
+				},
 				headerStyle: completionHeaderStyle,
 				columns: columns.slice(0, 7).map((column) => ({
 					...column,
+					sortingFn: sortEntityData,
+					header: (props) => {
+						const val = props.header.id;
+						const isThickBorder = val === 'FO';
```

Comment:
I assume this one is thick border cause its the last element? Should the logic be based on it being the last element not a specific element?

Reply:
It's somewhat unclear in regards to the groupings in the table.
Last element requires us to target the last element of the row.

We can't access the row element.
So we'd have to do some logic with the indices from the internal react table data which we shouldn't be touching at this level. That is way more flaky than just hard coding a value. 

DX wise `val === "FO` is very clear. It works and is very much in line with the code quality in the rest of this file

Reply:
Sounds fine, it is clear whats happening. Just to complete my though there though, do we not have access the index form the .map? partial code example:

```ts
const coreCompletionColumnsCount = 7;
columns: columns.slice(0, coreCompletionColumnsCount).map((column, index) => ({
	...column,
	sortingFn: sortEntityData,
	header: (props) => {
		const isLastElement = index === coreCompletionColumnsCount -1;

```